amundsen_application/static/js/components/TableDetail/ReportTableIssue/index.spec.tsx

```diff
@@ -92,9 +92,9 @@ describe('ReportTableIssue', () => {
         const { wrapper } = setup();
         const previsOpenState = wrapper.state().isOpen;
 
-        wrapper
-          .instance()
-          .toggle({ currentTarget: { id: 'id', nodeName: 'button' } });
+        wrapper.instance().toggle({
```

Comment:
Notice the test was updated but not the component. No issue though but as mentioned looks like we need to go through and formally update to using `dataset.type` for logging the location of a particular interaction we're interested in.

Reply:
Not sure I understand, I updated 'TableDetail/FrequentUsers/index.tsx' with a data attribute.

Reply:
And this is the `TableDetail/ReportTableIssue` test. :)

Reply:
Ohhhh.... bringing this back, thanks!!